Add TR1 secret locations

lahm86 opened the pull request:
A handful of new secrets for TR1. Committing before I forget about them.
Part of #510.

chreden reviewed (approved):
I assume it's OK that they aren't all Lahm PackID (and one secret lost a PackID)? Hard to tell without context.

lahm86 commented:
> I assume it's OK that they aren't all Lahm PackID (and one secret lost a PackID)? Hard to tell without context.

Yes, so for packs there should be a maximum of 5 defined per level (tied to trigger flags for TR1 - we only add one door to reward rooms). The first one added here is a nicer secret in Lost Valley than the one where I removed the pack ID, so it takes its place - Lost Valley had 5 defined already. Cistern had only 3 defined, but the new ones I've added here all require the same technique to get them, so only one makes it into the pack (the one that happens to be harder than the others). So if that level is allocated 5 secrets in pack mode, the last one will be random.
